src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -4269,6 +4269,34 @@ class C1
                 );
         }
 
+        [WorkItem(22813, "https://github.com/dotnet/roslyn/issues/67518")]
+        [Fact]
+        public void TestDynamicOperationRefStructArgument()
+        {
+            string source = @"
+static class Program
+{
+    static void Main()
+    {
+        dynamic d = 0;
+        R x = new R(d);
+        R y;
+        y = new R(default(R), d);
+    }
+}
+
+readonly ref struct R
+{
+    public R(int i) { }
+    public R(R r, int i) { }
+}
+";
+            CreateCompilationWithMscorlib40AndSystemCore(source, parseOptions: TestOptions.Regular7_2).VerifyDiagnostics(
+                // (9,19): error CS1978: Cannot use an expression of type 'R' as an argument to a dynamically dispatched operation
```

Comment:
>// (9,19): error CS1978: Cannot use an expression of type 'R' as an argument to a dynamically dispatched operation [](http://example.com/codeflow?start=16&length=114)

Please include the source code line into the comment as well. Like the previous test does, for example.